src/uct/ib/ud/base/ud_ep.c

```diff
@@ -556,18 +560,22 @@ ucs_status_t uct_ud_ep_create_connected_common(const uct_ep_params_t *ep_params,
 
     status = uct_ud_ep_connect_to_iface(ep, ib_addr, if_addr);
     if (status != UCS_OK) {
-        goto out;
+        goto err_disconnect;
```

Comment:
why need disconnect here? Also this func always returns `UCS_OK` anyway

Reply:
rename to "err_unbind_from_iface"
here "disconnect" means unbinding from iface

Reply:
> Also this func always returns UCS_OK anyway

we return `status` instead of `UCS_OK`